Code/Framework/AzFramework/AzFramework/Application/Application.cpp

```diff
@@ -391,6 +396,7 @@ namespace AzFramework
     {
         AZ::ComponentApplication::CreateStaticModules(outModules);
 
+        outModules.emplace_back(aznew AzNetworking::AzNetworkingModule());
```

Comment:
We should look into initializing the AzNetworking library as a gem or a SHARED library so that AzFramework doesn't need to depend on it.

Reply:
SHARED library sounds like the right approach. I'll take a look at that.

Reply:
Tools gem ;) it makes absolutely all of this go away. I've done a quick audit of the target manager and script debugging hooks inside azframework and all of it looks to be easily sliceable.. no other aspects of azframework appear to depend on this functionality.

Reply:
I might be able to revisit moving to AzToolsFramework. The main issue is Launchers need TargetManagement so they can connect to LuaIDE but now that it's compiled out in RELEASE, maybe this can be made to work.

On another note, from what I'm seeing there needs to be a single registration point for NetworkSystemsComponent so even with a Tools Gem the registration would need to be solved.